ClientQuickstart/Controllers/TokenController.cs

```diff
@@ -20,10 +22,13 @@ public ActionResult Index()
       var identity = Internet.UserName().AlphanumericOnly();
 
       // Create an Access Token generator
-      var capability = new TwilioCapability(accountSid, authToken);
-      capability.AllowClientOutgoing(appSid);
-      capability.AllowClientIncoming(identity);
-      var token = capability.GenerateToken();
+      var scopes = new HashSet<IScope>
+      {
+          { new IncomingClientScope(identity) },
+          { new OutgoingClientScope(appSid) }
+      };
+      var capability = new ClientCapability(accountSid, authToken);
```

Comment:
This should be:
     var capability = new ClientCapability(accountSid, authToken, scopes: scopes);
Gotta pass in the scopes